src/main/kotlin/no/nav/familie/ef/mottak/repository/SøknadDAO.kt

```diff
@@ -0,0 +1,57 @@
+package no.nav.familie.ef.mottak.repository
+
+import no.nav.familie.ef.mottak.repository.domain.Søknad
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
+import org.springframework.stereotype.Repository
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.sql.ResultSet
+import org.springframework.jdbc.support.GeneratedKeyHolder
+
+@Service
+@Repository
+@Transactional
+class SøknadDAO(private val namedParameterJdbcTemplate: NamedParameterJdbcTemplate) {
```

Comment:
![bitmoji](https://render.bitstrips.com/v2/cpanel/fbf70eb2-c012-465e-9c1e-7b092675dfd8-2023545f-5427-4c51-a633-2f0310070c6f-v1.png?transparent=1&palette=1&width=246)

Reply:
Update etter Mattis har utforska litt:
Ser ut som vi kan få en nokså red klasse med CrudRepository likevel, men uten confen i `ApplicationConfig.kt`. Kan ser på det sammen :)

Reply:
Jepp. Alt som skal til er: 
interface SoknadRepository : CrudRepository<Soknad, Long> {
    @Query("select * from soknad where id = :id")
    fun findSoknadById(id: Long) : Soknad
       override fun findAll() : List<Soknad>
}

Vi må fortsatt diskutere/finne ut hvordan vi gjør det med f.eks id på nye objekter @bent-lorentzen har helt sikkert noen tanker om dette :)

Reply:
PS. Måtte endre navn på Søknad til Soknad, men det finnes sikkert en bedre måte å gjøre dette på.

Reply:
Ser ut til at dere har prøvd å skrive mye kode for å gjenskape funksjonalitet som allerede ligger i master. Jeg har laget en test-branch nå som heter databasetest. Den lagrer og henter søknader fra en lokalt kjørende postgreSql-base på min maskin (Fikk ikke docker til å virke i dag). Den branchen burde gi litt pekepinn i riktig retning. Det den ikke har er spring cloud vault bootstrap-logikken. Den kjenner jeg ikke så godt. 

Jeg tok utgangspunkt i master fordi veldig mye av koden som er fjernet i denne branchen er i bruk, og kommer sannsynligvis til å være det.

Reply:
Nice, da tar jeg en titt på den branchen! Det er egentlig bare DAO som gjenskaper repository, men siden jeg bare kommenterte ut eksisterende metoder i stedet for å bare bytte ut parametre etc så ser det sikkert ut som mye.

Det kom nok ikke så tydelig fram, men med "ubrukt" så tenker jeg ikke at vi ikke kommer til å bruke det, men siden oppgaven her dreier seg om å få på plass lagring og ikke oppgavehåndtering kommenterte jeg ut kode som enn så lenge det står "standby". Vi får sikkert behov for det når vi begynner å jobbe med det :) Kanskje blir det da naturlig å koble vedlegg og søknad på en henvendelse. Har ikke lagt så mye tid i datastruktur da jeg håper dette er noe vi kan se på i fellesskap.